samza-core/src/main/scala/org/apache/samza/coordinator/JobCoordinator.scala

```diff
@@ -121,8 +121,9 @@ object JobModelManager extends Logging {
                         changelogManager: ChangelogPartitionManager,
                         localityManager: LocalityManager,
                         streamMetadataCache: StreamMetadataCache,
-                        streamPartitionCountMonitor: StreamPartitionCountMonitor) = {
-    val jobModel: JobModel = initializeJobModel(config, changelogManager, localityManager, streamMetadataCache)
+                        streamPartitionCountMonitor: StreamPartitionCountMonitor,
+                        containerIds: java.util.List[Integer]) = {
```

Comment:
Ah.. This seems to have trickled from a future patch :) 
When we are operating using ZK for coordination, we will have an elastic set of processing nodes. So, containerIds , aka processorIds are needed for JobModel generation (since it is no longer between 0 and N-1). 

Since this change is not directly related to the current JIRA, I will remove it. Thanks for noticing it!